artisan.plugin.zsh

```diff
@@ -7,37 +7,55 @@
 # adds shell completions that work anywhere artisan can be located.
 
 function artisan() {
-    _artisan=`_artisan_find`
+    local artisan_path=`_artisan_find`
 
-    if [ "$_artisan" = "" ]; then
+    if [ "$artisan_path" = "" ]; then
         >&2 echo "zsh-artisan: You seem to have upset the delicate internal balance of my housekeeper."
         return 1
     fi
 
-    _artisan_start_time=`date +%s`
-    php $_artisan $*
-    _artisan_exit_status=$? # Store the exit status so we can return it later
+    local laravel_path=`dirname $artisan_path`
+    local docker_compose_config_path=`find $laravel_path -maxdepth 1 -regex ".*/docker-compose.ya?ml" | head -n1`
```

Comment:
To get this working on macOS, I had to modify this to `find -E $laravel_path` to use extended regular expressions (ERE). The issue was around the `a?` in searching for yaml/yml file types. I don't know if that breaks anything in your machine though.

This is the page that pointed me in the right direction: https://unix.stackexchange.com/questions/479298/make-find-regextype-egrep-as-alias

Reply:
Sorry for the massive delay.

Unfortunately, `-E` gives this error with GNU find:

> find: unknown predicate `-E'

Reply:
Instead of using the regex, I've changed it to use `-o` and `-name` to search for multiple names. Hopefully, that fixes it on macOS.